### PR TITLE
Updating a variant clickable for cards

### LINKED ISSUE #58: Assign an anchor tag to the entire cards block so that we can click it
image

### PR DESCRIPTION
**Pull request by meejain**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #58 

Test URLs:
- Before: https://main--clarkcountynv--aemsites.hlx.page/drafts/meet/doc1
- After: https://issue58-clickable--clarkcountynv--aemsites.hlx.page/drafts/meet/doc1

### CODE REVIEW

File: blocks/cards/cards.js [hunk 1]
```diff
@@ -4,13 +4,31 @@ export default function decorate(block) {
...
   [...block.children].forEach((row) => {
-    const li = document.createElement('li');
-    while (row.firstElementChild) li.append(row.firstElementChild);
-    [...li.children].forEach((div) => {
-      if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
-    });
-    ul.append(li);
+    if (block.classList.contains('clickable')) {
+      const li = document.createElement('li');
```
Comment (akasjain2) [inline blocks/cards/cards.js:8]: Having integrated dom-helpers, we can leverage them to keep the code shorter.

const li = li();
const aEle = a(li);

**Review by akasjain2 — APPROVED** [review]
LGTM, left a minor comment.

**Review by klivingston — APPROVED** [review]
Nice, looks good

File: blocks/cards/cards.js (hunk 1 above)
Comment (meejain) [reply blocks/cards/cards.js:8]: This is Ack'ed @akasjain2

**Pull request merged**